ui/src/routes/settings/index.tsx

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (C) 2025 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+import { zodResolver } from '@hookform/resolvers/zod';
+import { createFileRoute } from '@tanstack/react-router';
+import { useForm } from 'react-hook-form';
+import { z } from 'zod';
+
+import { Button } from '@/components/ui/button';
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardHeader,
+} from '@/components/ui/card';
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from '@/components/ui/form';
+import { RadioGroup, RadioGroupItem } from '@/components/ui/radio-group';
+import { Separator } from '@/components/ui/separator';
+import { toast } from '@/lib/toast';
+import { packageIdTypeSchema } from '@/schemas';
+import { useUserSettingsStore } from '@/store/user-settings.store';
+
+const formSchema = z.object({
+  packageIdType: packageIdTypeSchema,
+});
+
+const SettingsPage = () => {
+  const packageIdType = useUserSettingsStore((state) => state.packageIdType);
+  const setPackageIdType = useUserSettingsStore(
+    (state) => state.setPackageIdType
+  );
+
+  const form = useForm<z.infer<typeof formSchema>>({
+    resolver: zodResolver(formSchema),
+    defaultValues: {
+      packageIdType: packageIdType,
+    },
+  });
+
+  async function onSubmit(values: z.infer<typeof formSchema>) {
+    setPackageIdType(values.packageIdType);
+    toast.info('User Settings', {
+      description: `User settings saved to browser.`,
+    });
+  }
+
+  return (
+    <Card className='mx-auto w-full max-w-4xl'>
+      <CardHeader>
+        <h2 className='text-3xl font-bold tracking-tight'>User Settings</h2>
+        <CardDescription>
+          Here you can configure your own user preferences for ORT Server. The
+          preferences will be saved to your browser, so please note that
+          clearing your browser cache will remove them and return the default
+          preferences.
```

Comment:
Wording-wise I believe we should not mix "settings" and "preferences", but stick to either. As the page is called "User Settings", I'd stick to "settings" as the term.

Also passive voice should be used, not directly talking to the user. I propose:

"These user-specific settings are saved as cookies in the browser. Clearing browser cookies will result in settings to be reset to their defaults."